CHANGELOG.md

```diff
@@ -28,6 +28,7 @@
 - revised tech selections route to update only one tech per request([#221](https://github.com/chingu-x/chingu-dashboard-be/pull/221))
 
 ### Fixed
+- fixed POST voyages/teams/{teamId}/techs bug , verify that categoryId is owned by correct team ([#228](https://github.com/chingu-x/chingu-dashboard-be/pull/228))
```

Comment:
I think these top categories are part of the Unreleased section - is that wrong?

Reply:
I think you'll need to update branch then it will change because I did a release after you submitted this PR

![image](https://github.com/user-attachments/assets/db7e6227-92fe-4668-b40a-25fb65679074)

Reply:
Ok I updated the new changelog